apps/desktop-wallet/src/components/AddressBadge.tsx

```diff
@@ -78,15 +81,20 @@ const AddressBadge = ({
         <>
           {!hideColorIndication && <AddressColorIndicator addressHash={address.hash} hideMainAddressBadge={hideStar} />}
           {address.label ? (
-            <Label truncate={truncate}>
-              {disableCopy ? (
-                address.label
-              ) : (
-                <ClipboardButton textToCopy={address.hash} tooltip={t('Copy address')} disableA11y={disableA11y}>
-                  {address.label}
-                </ClipboardButton>
+            <LabelAndHash column={displayHashUnder}>
```

Comment:
nit: At first I thought the `column` prop expects a number, how about renaming it to `isColumn`?